handlers/handlers.go

```diff
@@ -1,194 +1,127 @@
 package handlers
 
 import (
-	"time"
+	"patches/models"
+
 	"encoding/json"
 	"log"
 	"net/http"
-	"database/sql"
 	"io/ioutil"
+	"fmt"
+	"time"
 
-    _ "github.com/go-sql-driver/mysql"
-	_ "github.com/mattn/go-sqlite3"
+	"github.com/gorilla/schema"
 )
 
-type Patch struct {
-	Timestamp   time.Time      		 `json:"timestamp,omitempty"`
-	Patch 		string 				 `json:"patch,omitempty"`
-	Convo_ID    string               `json:"convo_id,omitempty"`
-	User_ID     string         		 `json:"user_id,omitempty"`
-	Type 		string 				 `json:"type,omitempty"`
+type Env struct {
+	DB models.Datastore
 }
 
-// Connect to database
-func dbConnect() (*sql.DB, error) {
-	db, err := sql.Open("sqlite3", "../patches.db")
-	if err != nil {
-		log.Fatal(err)
-		return db, err
-	}
-	// test connection
-	err = db.Ping()
-	if err != nil {
-		log.Fatal(err)
-		return db, err
-	}
-	return db, nil
+type Filter struct {
+    Conversation   	[]string 		`schema:"convoID"`
```

Comment:
These struct tags should be snake case instead of camel case to keep it consistent with the JSON fields of the other request bodies.